yo/db.py

```diff
@@ -321,7 +320,7 @@ def get_user_transports(self, username):
                 user_settings_table.c.username == username)
                 select_response = conn.execute(query)
                 results = select_response.fetchone()
-                if results:
```

Comment:
Enjoy python! `select_response.fetchone()` will either return a `RowProxy` (not False) class or `None` (False)

Reply:
That should be the case, but during testing it simply didn't work - I didn't want to waste time digging into why.